idm/pkg/config/config.go

```diff
@@ -16,15 +16,23 @@ type Config struct {
 	Log     *Log     `ocisConfig:"log"`
 	Debug   Debug    `ocisConfig:"debug"`
 
-	IDM Settings `ocisConfig:"idm"`
+	IDM             Settings `ocisConfig:"idm"`
+	CreateDemoUsers bool     `ocisConfig:"create_demo_users" env:"IDM_CREATE_DEMO_USERS;ACCOUNTS_DEMO_USERS_AND_GROUPS"`
+
+	ServiceUserPasswords ServiceUserPasswords `ocisConfig:"service_user_passwords"`
 
 	Context context.Context
 }
 
 type Settings struct {
-	LDAPSAddr     string `ocisConfig:"ldaps_addr" env:"IDM_LDAPS_ADDR"`
-	Cert          string `ocisConfig:"cert" env:"IDM_LDAPS_CERT"`
-	Key           string `ocisConfig:"cert" env:"IDM_LDAPS_KEY"`
-	DatabasePath  string `ocisConfig:"database" env:"IDM_DATABASE_PATH"`
-	AdminPassword string `ocisConfig:"admin_password" env:"IDM_ADMIN_PASSWORD"`
+	LDAPSAddr    string `ocisConfig:"ldaps_addr" env:"IDM_LDAPS_ADDR"`
+	Cert         string `ocisConfig:"cert" env:"IDM_LDAPS_CERT"`
+	Key          string `ocisConfig:"cert" env:"IDM_LDAPS_KEY"`
+	DatabasePath string `ocisConfig:"database" env:"IDM_DATABASE_PATH"`
+}
+
+type ServiceUserPasswords struct {
```

Comment:
Would you mind adding explaining comments to `Settings` and `ServiceUserPasswords` structs?

Reply:
done